server/service/user.go

```diff
@@ -129,3 +129,19 @@ func (u *UserService) GetUsers(page int, limit int) *view.Response {
 
 	return view.SuccessAllUsersResponse(query, payload)
 }
+
+func (u *UserService) ShowUserByEmail(email string) *view.Response {
+	user, err := u.repo.FindUserByEmail(email)
+	if err != nil {
+		log.Printf("Error when try to get user by email %v\n", err)
+		return view.ErrorResponse("GET_USER_BY_EMAIL_FAIL", "NOT_FOUND", http.StatusNotFound)
+	}
```

Comment:
ini buat error not found ya? mungkin bisa diimprove kayak gini mas @OxiCuza 
![image](https://user-images.githubusercontent.com/57651747/198834038-531beec9-524d-46ef-90c7-1cc10ada3de9.png)